ideascube/library/static/library/library.css

```diff
@@ -0,0 +1,42 @@
+/* ************************************************* */
```

Comment:
While a fully understand why you made this move, it may have unwanted side effects: we are trying to minimize the number of requests run in parallel, in order to let the Koombook handle more users.
Given that the static files are cached, the split may have no impact, but I think we should balance this and make sure we are not doing a mistake here. :)